runtime/compiler/env/PersistentCHTable.cpp

```diff
@@ -445,8 +445,8 @@ TR_PersistentCHTable::hasTwoOrMoreCompiledImplementors(
    TR_PersistentClassInfo * classInfo = findClassInfoAfterLocking(thisClass, comp, true);
    if (!classInfo) return false;
 
-   TR_ResolvedMethod *implArray[2];
-   return TR_ClassQueries::collectCompiledImplementorsCapped(classInfo,implArray,2,cpIndex,callerMethod,comp,hotness,locked) == 2;
```

Comment:
The method is called hasTwoOrMoreCompiledImplementors so the name needs to change if it is becoming 3 or more :)

Reply:
Good catch. I'm fixing that right now.